### DISCUSSION TITLE
Criar um campo de categorias para os produtos

### DISCUSSION THREAD

**Discussion by rayhenrique** [RamonSilva20/mapos#1815] [Ideas]
Parabéns pelo projeto, inclusive estou usando o mesmo para gerenciar o almoxarifado e setor de T.I. da secretaria que trabalho.

Tem como inserir o campo "categoria" no cadastro de produtos e ao mesmo tempo inserir isso nos relatório?

Desde já, obrigado!

**Comment by FlaviusR** [comment]
Interessante. mas, acho que já vi por aí. Alguém que implementou categoria e fotos dos produtos.

**Comment by rayhenrique** [comment]
Gostaria muito desse recurso... :/

Em sex., 13 de mai. de 2022 às 12:07, Flavio Renato <
***@***.***> escreveu:

**Comment by SuperPorte** [comment]
Isso e o sistema salvar o margem de lucro dos produtos.
Sempre que reponho estoque com preço diferente eu tenho de lembrar a margem dos itens.
